src/client/identity/identity.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package identity;
+option go_package = "github.com/pachyderm/pachyderm/src/client/identity";
```

Comment:
The identity API is basically a more pachyderm-idiomatic version of the dex grpc API. One addition is the ability to configure IDP connectors - we should be able to upstream this but it may happen post 2.0.